src/lib/orysdk/index.ts

```diff
@@ -3,7 +3,6 @@ import { edgeConfig } from "@ory/integrations/next";
 
 let oryConfig = new Configuration(edgeConfig);
 
-if (process.env.NEXT_PUBLIC_ORYCLOUD === "enabled") {
     oryConfig = new Configuration({
```

Comment:
issue(blocking): This line will break the production environment, please restore it.

Reply:
They were resolved, @caneppelevitor  helped me, he said it was probably a problem with a git pull that I did wrong, since I hadn't even touched those files, I only open them now that I had to resolve it, sorry for the inconvenience